include/revng/Pipeline/Invokable.h

```diff
@@ -392,9 +393,10 @@ concept Printable = requires(InvokableType Pipe) {
 
 class InvokableWrapperBase {
 public:
-  virtual void run(Context &Ctx,
-                   ContainerSet &Containers,
-                   const llvm::StringMap<std::string> &ExtraArgs = {}) = 0;
+  virtual void 
+  run(Context &Ctx,
+      ContainerSet &Containers,
+      const llvm::StringMap<std::string> &ExtraArgs = {}) = 0;
```

Comment:
Uh? Same?
Did you run `orc check-branch`?

Reply:
some what? not sure what this refers to

Reply:
The changeset seems to be just reformatting the code, which should not happen if `revng-check-conventions` passes on all commits.